src/WebJobs.Script/Config/WorkerConcurrencyOptions.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class WorkerConcurrencyOptions : Hosting.IOptionsFormatter
+    {
+        public WorkerConcurrencyOptions()
+        {
+            // Setting deafault values
+            LatencyThreshold = TimeSpan.FromSeconds(1);
+            AdjustmentPeriod = TimeSpan.FromSeconds(10);
+            CheckInterval = TimeSpan.FromSeconds(1);
+            HistorySize = 10;
+            HistoryThreshold = 1F;
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether worker concurrency
```

Comment:
I think in user doc we should refer to this as "worker dynamic concurrency" to make it clear that we're talking about dynamic adjustment of worker counts